RoeiVerenigingWPF/Pages/Member/ListEvents.xaml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using DataAccessLibrary;
+using RoeiVerenigingLibrary.Model;
+using RoeiVerenigingLibrary.Services;
+using RoeiVerenigingWPF.Frames;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+namespace RoeiVerenigingWPF.Pages.Member
+{
+    /// <summary>
+    /// Interaction logic for ListEvents.xaml
+    /// </summary>
+
+    public partial class ListEvents : Page
+    {
+
+        public List<Event> _events { get; set; }
```

Comment:
dit moet EventList wordt of iets in die richting, moet sws met hoofdletter want het is een public property